lighthouse-core/test/lib/i18n/i18n-test.js

```diff
@@ -191,5 +196,15 @@ describe('i18n', () => {
         // eslint-disable-next-line max-len
         .toThrow(`ICU Message "Hello {timeInMs, number, seconds} World" contains a numeric reference ("timeInMs") but provided value was not a number`);
     });
+
+    it('throws an error if a value is provided that has no placeholder in the message', () => {
+      const helloStr = str_(UIStrings.helloTimeInMsWorld, {
+        timeInMs: 55,
+        sirNotAppearingInThisString: 66,
+      });
+      expect(_ => i18n.getFormatted(helloStr, 'en-US'))
+        // eslint-disable-next-line max-len
```

Comment:
apparently we've decided there's disagreement on this point so we should never use `disable-next-line max-len` again :P

@exterkamp I'm gonna love this convo next week where I can learn how splitting is possibly better 😆